data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -68,11 +68,64 @@ local whole_disk_and_boot_unattended() = {
     device: 'boot-disk',
   },
 };
+
+local raid0() = {
+  drives: [
+    {
+      search: { max: '4' },
```

Comment:
as we have fixed NUMDISK=4 in openQA we don't need this I think.

Reply:
yes, removed it, thanks.